internal/parser/snippet.go

```diff
@@ -94,3 +99,31 @@ func (s Snippets) Names() (result []string) {
 	}
 	return result
 }
+
+func (s *Snippet) extractParameters() []string {
+	match := parameterRe.FindAllStringSubmatch(s.content, -1)
+	var parameters []string
+	for _, m := range match {
+		parameters = append(parameters, m[0])
+	}
+	return parameters
+}
+
+func (s *Snippet) mapParameterValues(parameters []string, values []string) string {
+	var paramExp regexp.Regexp
+	newCmd := s.Content()
+	for i := range parameters {
```

Comment:
```suggestion
	for i := range parameters {
	    newCmd = strings.ReplaceAll(newCmd, parameters[i], values[i])
	}
```

That should work, I don't see a reason we need to use `regexp` package here.